src/generateRandomNumber.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function generateRandomNumber() {
```

Comment:
The logic for generating random numbers is too complicated, since we first get the index, then we take this index from the array, add it to the number, while removing this number from the array. Wouldn't it be easier to just take a randomly generated number and put it in a `Set` until the length of this `Set` is 4?